.github/workflows/build.yaml

```diff
@@ -108,7 +108,8 @@ jobs:
 
       - name: Upload Spectron Logs
         uses: actions/upload-artifact@v2
-        if: ${{ cancelled() || failure() }}
+        # even if tests succeeded you might still want to examine logs
+        if: always()
```

Comment:
```suggestion
```
Small thing, do these run still run without `if: always`? Maybe we can just remove them.

Reply:
Without `always` it will not happen on `cancelled` or `failure`. By default job steps happen only if the previous one didn't fail

Reply:
Ah gotcha thanks!

Reply:
Good that you jumped in because I was about to make the suggested change ;)